### DISCUSSION TITLE
Is it possible to get text streamed into a python script?

### DISCUSSION THREAD

**Discussion by hoonlight** [waylaidwanderer/node-chatgpt-api#190] [Help] [answered]
Hi, I read more about getting responses sent with SSE.

However, since I have no knowledge of js and I am making a bot in python, I have to implement this in python.

How can I implement this?

https://github.com/waylaidwanderer/node-chatgpt-api/blob/main/demos/use-api-server-streaming.js

Even reading this, I'm not sure. help..

**Comment by hoonlight** [comment, marked as answer]
I solved it using aiohttp.

**Reply by waylaidwanderer** [reply]
Glad you figured it out. I don't have Python so I had no idea :P

**Reply by vaibhavard** [reply]
I had to again work my own way around text streaming:
Something like this ?
import requests
# Replace this with your own message
data = {
    "message": "What did i just ask you?",
    'jailbreakConversationId':True, 
    "stream": True,

}

# Replace this with your own ChatGPT API server URL
url = "http://localhost:3000/conversation"
with requests.post(url, json=data, stream=True) as resp:
        for line in resp.iter_lines():
            if line:
                print(line)
